packages/admin-panel/src/pages/StrivePage.jsx

```diff
@@ -21,24 +19,14 @@ const StyledBody = styled(PageBody)`
   padding-top: 2rem;
 `;
 
-export const StrivePage = ({ getHeaderEl }) => {
-  const HeaderPortal = usePortalWithCallback(
-    <Header title="Strive" importConfig={importConfig} />,
-    getHeaderEl,
-  );
+export const StrivePage = () => {
   return (
-    <>
-      {HeaderPortal}
-      <StyledBody>
-        <Typography variant="h4" gutterBottom>
-          Import lab results or vector data
-        </Typography>
-        <Typography>Use the above Import button to import lab results or vector data.</Typography>
-      </StyledBody>
-    </>
+    <StyledBody>
+      <PageHeader title="Strive" importConfig={importConfig} />
+      <Typography variant="h4" gutterBottom>
+        Import lab results or vector data
+      </Typography>
+      <Typography>Use the above Import button to import lab results or vector data.</Typography>
```

Comment:
Nitpicky Material Design style guide detail

```suggestion
      <Typography>Use the above Import button to import lab results or vector data</Typography>
```